halo2_gadgets/src/ecc/chip/mul_fixed/short.rs

```diff
@@ -519,39 +519,39 @@ pub mod tests {
                 // 2^64
                 MyCircuit {
                     magnitude: Value::known(pallas::Base::from_u128(1 << 64)),
-                    sign: Value::known(pallas::Base::one()),
+                    sign: FieldValue::ONE,
                     magnitude_error: Value::known(pallas::Base::from(1 << 1)),
                 },
                 // -2^64
                 MyCircuit {
                     magnitude: Value::known(pallas::Base::from_u128(1 << 64)),
-                    sign: Value::known(-pallas::Base::one()),
+                    sign: -Value::<pallas::Base>::ONE,
```

Comment:
However, situations like this require type annotations (to know which `impl Neg` to use), so you save a bit of length overall but it's still roughly as complex (if not moreso).

Reply:
I think adding the need of the explicit type requirement makes it worse TBH. Although `-1` is also one of the "magic numbers" usually. But I kinda agree that should not have it's own constant..

Reply:
Why not (have its own constant)? IIRC, -1 was *the* most common constant in the Orchard circuit.

Reply:
Yes I was remembering correctly: https://github.com/zcash/halo2/issues/698#issuecomment-1332365178

Reply:
> Why not (have its own constant)?

Because (at least in this PR) it would require 5 extra characters:

```rust
let a = -Value<pallas::Base>::ONE;
let a = Value<pallas::Base>::MINUS_ONE;
```

Reply:
> Why not (have its own constant)?

For -1 specifically it would help:

```rust
let a = -Value::<pallas::Base>::ONE;
let a = FieldValue::MINUS_ONE;
```

But the point still applies generally.